libraries/clocker.rb

```diff
@@ -0,0 +1,116 @@
+class Clocker
+  class ZKs
+    def initialize
+      # hash of zookeeper locks we use one connection per lock id
+      @@zkls = {}
+      return
+    end
+
+    # Create or retrieve a locker as appriate
```

Comment:
thanks for spotting all my :space_invader: typos, this is what happens when I use a spell checker!